packages/wonder-blocks-pill/package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "@khanacademy/wonder-blocks-pill",
+    "version": "0.0.1",
+    "design": "v1",
+    "description": "Pill components for Wonder Blocks.",
```

Comment:
**suggestion:** This config should include the following settings to avoid releasing the newly created package along other packages. Of course, we'll switch to this version when `Pill` is ready for PROD (or `v1.0.0`)

```suggestion
    "description": "Pill components for Wonder Blocks.",
    "private": true,
    "publishConfig": {
        "access": "restricted"
    },
```